include/cppad/cg/code_handler.hpp

```diff
@@ -293,7 +293,7 @@ class CodeHandler {
      * @return a pointer to the atomic function name if it was registered
      *         or nullptr otherwise
      */
-    inline const std::string* getAtomicFunctionName(size_t id) const;
```

Comment:
Please see the changes in the branch feature/parameters:
https://github.com/joaoleal/CppADCodeGen/blob/feature/parameters/include/cppad/cg/code_handler.hpp

I went with:
`inline std::unique_ptr<std::string> getAtomicFunctionName(size_t id) const;`
so that I could easily identify undefined atomic functions but the name cannot be empty and therefore your approach is actually better.
We just need to update the documentation (`@return the atomic function name if it was registered or an empty string otherwise`).